### DISCUSSION TITLE
TextViews show black text on gray background in Android in Dark mode

### DISCUSSION THREAD

**Discussion by bear101** [BearWare/TeamTalk5#1699] [Q&A]
Hi @beqabeqa473

On Android 11 in Dark mode the server list activity appears with black text on a dark gray background making it almost impossible to read the text. As far as I can tell we don't change any colors manually, so how come Android cannot default to the correct colors?

Do you have any idea on how to solve this? When I googled this I could only find way where people created their own themes but that seems a bit excessive.

**Comment by beqabeqa473** [comment]
Hi.

Unfortunately, because of impossibility to test changes, i am not doing anything in terms of visual feedback.

I know that we should derive from daynight theme. but i never dived into that.

I will try to investigate this thing with help from sighted person.
